documentation/components/tutorial-flow-components-setup.asciidoc

```diff
@@ -139,9 +109,29 @@ When using Maven you can add a the button component into your `pom.xml` as this:
 </dependencies>
 ----
 
-== Maven configuration to exclude webjars
+== Using WebJars
+
+Individual component packages consists of two main sections:
+
+. Java API classes.
+. Web Component files (HTML, JavaScript, CSS ..).
+
+Both sections are provided in the same JAR by default, and both sections are used to display the components correctly in pages.
+
+Web Component files are provided as https://github.com/webjars/webjars/[WebJars] (JAR files that contain Web Component files), the design of which was heavily influenced by the Polymer package manager, https://bower.io/[Bower] (not developed or maintained by Vaadin). WebJars facilitate using Maven, instead of Bower, for the same purpose. 
 
-If webjar dependencies are not needed, they can be excluded from the project using standard Maven mechanism:
+Flow is able to resolve requests into WebJar-compatible content by imitating regular Web Component files.
```

Comment:
I'd just remove this line altogether. Flow is not imitating anything and there are no "regular Web Component files", but Flow just serves the client side requests from the archive when asked.

```suggestion
```